### PR TITLE
Inconsistent & wrong behaviour of the button to reactivate exposure logging on the risk card in different scenarios

### PR DESCRIPTION
**Pull request by freshking**
## Description

Fix inconsistent behaviour surrounding exposure notification settings and bluetooth.

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5579

### CODE REVIEW

**Comment by Ein-Tim** [comment]
Fixes #2157.

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 1]
```diff
@@ -361,7 +361,6 @@ enum AppStrings {
 		static let lowRiskExposureSubtitle = NSLocalizedString("ExposureDetection_LowRiskExposure_Subtitle", comment: "")
 		static let lowRiskExposureBody = NSLocalizedString("ExposureDetection_LowRiskExposure_Body", comment: "")
 
-		static let buttonEnable = NSLocalizedString("ExposureDetection_Button_Enable", comment: "")
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:364]: Is `ExposureDetection_Button_Enable` still used somewhere else or can it be removed from the strings file as well?

Comment (freshking) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:364]: I removed it from the strings file as well

**Review by marcussc — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA reviewed.

**Pull request merged**